.env.example

```
@@ -18,6 +18,12 @@ SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:5432/예시_데이터
 SPRING_DATASOURCE_USERNAME=예시_데이터
 SPRING_DATASOURCE_PASSWORD=예시_데이터
 
+# Load Test용 Datasource
+SPRING_DATASOURCE_URL_LOADTEST=예시_데이터
+SPRING_DATASOURCE_USERNAME_LOADTEST=예시_데이터
+SPRING_DATASOURCE_PASSWORD_LOADTEST=예시_데이터
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SPRING_DATASOURCE_PASSWORD_LOADTEST key should go before the SPRING_DATASOURCE_URL_LOADTEST key

(UnorderedKey)